src/home/DappsCarousel.tsx

```diff
@@ -149,7 +149,12 @@ function DappsCarousel({ onSelectDapp }: Props) {
                   resizeMode="cover"
                   testID={`${testID}/Icon`}
                 />
-                <Text style={styles.dappName} numberOfLines={1} ellipsizeMode="tail">
+                <Text
+                  style={styles.dappName}
+                  numberOfLines={1}
+                  ellipsizeMode="tail"
+                  testID="RecentDapp-name"
```

Comment:
nit: testID={`${testID}/name`}

Reply:
Changed in 2578442707407aa15bbeba6e1e55b3b213dae616!